fuel/transformers/__init__.py

```diff
@@ -590,3 +590,31 @@ def get_data(self, request=None):
         if data == StopIteration:
             raise StopIteration
         return data
+
+
+class RenameStream(Transformer):
+    """Renames the sources of the stream.
+
+    Parameters
+    ----------
+    data_stream : :class:`DataStream` or :class:`Transformer`
+        The data stream.
+    names : dict
+        A dictionnary mapping the old and new names of the sources to rename.
```

Comment:
This is too long, according to PEP8. Break that in two lines of < 75 characters, it'll fix the Scrutinizer failure.